File: Entities/Multa.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Entities
{
    public class Multa : BaseEntity
    {
        public int id { get; set; }
        public int id_usuario { get; set; }
        public int id_item { get; set; }
        public int id_comercio { get; set; }
        public int id_sucursal { get; set; }
        public DateTime fecha { get; set; }
        public Usuario usuario { get; set; }
        public Item item { get; set; }
        public Comercio comercio { get; set; }
        public Sucursal sucursal { get; set; }

    }
}

File: Web_UI/ClientApp/src/app/models/multa.ts
import { Usuario } from "./usuario.model";
import { Item } from "./item";
import { Comercio } from "./Comercio";
import { Sucursal } from "./Sucursal";

export class Multa {
  id: number;
  id_usuario: number;
  id_item: number;
  id_comercio: number;
  id_sucursal: number;
  fecha: Date;
  usuario: Usuario;
  item: Item;
  comercio: Comercio;
  sucursal: Sucursal;
}
